theme/themes/eea/collections/table.variables

```diff
@@ -38,12 +42,12 @@
 @cellBorder: 1px solid @internalBorderColor;
 
 /* Table Header */
-@headerBorder: 1px solid @internalBorderColor;
+@headerBorder: 1px solid @secondaryColor;
 @headerDivider: none;
-@headerBackground: @headerWhite;
+@headerBackground: @silverGray;
 @headerAlign: inherit;
 @headerVerticalAlign: inherit;
-@headerColor: @lightGrey;
+@headerColor: @policeBlue;
```

Comment:
Can we have a better name for this color? We typically avoid "culture-sensitive" names.